src/RegisterService.Tests/Unit/OrdersController.Test.cs

```diff
@@ -0,0 +1,83 @@
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using ProdtestApi.Services;
+using ProdtestApi.Controllers;
+using ProdtestApi.Contracts;
+using ProdtestApi.Models;
+
+namespace ProdtestApi.Test.Unit;
+
+[TestClass]
+public class OrderControllerTests
+{
+    private Mock<IKitchenService> mock { get; } = new Mock<IKitchenService>();
```

Comment:
Volgens mij gebruik je op deze manier 1 mock voor alle testen? of wordt de class na iedere test opnieuw geinitialiseerd? zo niet, dan kunnen setups van verschillende testen elkaar in de weg zitten.